calico-cloud/get-started/connect/requirements/aks.mdx

```diff
@@ -16,7 +21,7 @@ Although {{prodname}} checks that your cluster meets [System requirements](syste
    az aks show --query 'networkProfile'
    ```
 
-   You must have one of the following combinations of values for`networkPlugin` and `networkPolicy`:
+  You must have one of the following combinations of values for`networkPlugin` and `networkPolicy`:
```

Comment:
This section seems to contradict what Faisal is saying. So only the first bullet is an option or CC cannot manage it? FYI, no one has touched this file since we migrated content from jekyll so maybe the doc is just wrong?


  You must have one of the following combinations of values for`networkPlugin` and `networkPolicy`:

   - `"networkPlugin": "none"` and `"networkPolicy": null`
   - `"networkPlugin": "azure"` and `"networkPolicy": null`
   - `"networkPlugin": "azure"` and `"networkPolicy": "calico"`